packages/ketcher-standalone/src/infrastructure/services/struct/standaloneStructServiceProvider.ts

```diff
@@ -0,0 +1,10 @@
+import { StructService, StructServiceProvider } from './structService.types'
+import StandaloneStructService from './standaloneStructService'
+
+class StandaloneStructServiceProvider implements StructServiceProvider {
+  initStructService(options: any): StructService {
+    ;(global as any).ketcher.standalone = true
```

Comment:
It should be executed once. Please move it into constructor

Reply:
done